### PR TITLE
feat(services): Creating POST api /postcards

### PR DESCRIPTION
**Pull request by shraddha1704**
### What
Creating a POST api `/postcards` endpoint

### Why
Angular training

### CODE REVIEW

**Review by mgartner — APPROVED** [review]

File: app/services/postcard.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = function (API) {
+
+  var Postcard = {};
+
+  Postcard.url = Config.API_HOST + '/postcards';
+
+  Postcard.create = function (payload) {
+    return API.post(this.url, payload);
+  }
```
Comment (kylekwong) [inline app/services/postcard.js]: nit: semicolon

File: app/services/postcard.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  return Postcard;
+
+}
```
Comment (kylekwong) [inline app/services/postcard.js]: nit: semicolon

File: app/services/postcard.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  var Postcard = {};
+
+  Postcard.url = Config.API_HOST + '/postcards';
+
+  Postcard.create = function (payload) {
+    return API.post(this.url, payload);
+  }
+
+  return Postcard;
+
```
Comment (kylekwong) [inline app/services/postcard.js]: nit: delete this newline

File: test/services/postcard.spec.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Config = require('../../config');
```
Comment (kylekwong) [inline test/services/postcard.spec.js]: nit: make this `=` aligned with all the other ones in this block

File: test/services/postcard.spec.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+describe('postcard service', function () {
+
+  var $q;
+  var $rootScope;
+  var API;
+  var Postcard;
+
+  beforeEach(Angular.mock.inject(function ($injector) {
+    $q = $injector.get('$q');
```
Comment (kylekwong) [inline test/services/postcard.spec.js]: nit: align all of these `=`

**Review by kylekwong — APPROVED** [review]
just nits

**Pull request merged**